lib/irc/ClientPool.js

```diff
@@ -261,10 +261,6 @@ ClientPool.prototype._onClientDisconnected = function(bridgedClient) {
         return;
     }
     // Reconnect this user
-    log.debug(
-        "onClientDisconnected: <%s> Reconnecting %s@%s in %sms",
-        bridgedClient._id, bridgedClient.nick, bridgedClient.server.domain, RECONNECT_TIME_MS
-    );
```

Comment:
Is this not a useful thing to have?

Reply:
The problem with it is that if we lose connection on a large box we spam this once per client connection, which is 22k times on freenode which tends to be deadly.

Reply:
Fair. So long as there is an indication that 22k clients are reconnecting.

Reply:
The hope is the queue func that logs waiting items above is good enough of an indicator.

Reply:
(I'm aware it will also spam a lot, but hopefully not quite as quickly so as to run out of resources)